CMakeLists.txt

```diff
@@ -188,7 +189,7 @@ endif()
 
 #Define what we want to be installed during a make install
 install(
-  TARGETS ${amr_wind_exe_name} ${amr_wind_unit_test_exe_name}
+  TARGETS amrex_hydro ${amr_wind_exe_name} ${amr_wind_unit_test_exe_name}
```

Comment:
Is there no qualified name similar to `AMReX::amrex` for the hydro libraries?

Reply:
@sayerhs I looked into it previously, and unless I missed it, unfortunately the hydro libraries don't appear to have a qualified name. Incflo seems to be using `amrex_hydro` too - https://github.com/AMReX-Codes/incflo/blob/development/CMakeLists.txt#L147

Reply:
May be something to request in AMReX-hydro package for future.

Reply:
@ashesh2512 Since `amrex_hydro` is an OBJECT library, I believe this should not be present. Have  you confirmed using this _installed_ `amr-wind` library in the exawind driver project?

Reply:
@sayerhs I have not built the exawind driver using the changes here, although without amrex_hydro as a target, the cmake process fails with the following errors - 

CMake Error: install(EXPORT "AMR-WindTargets" ...) includes target "amrwind_api" which requires target "amrex_hydro" that is not in any export set.
CMake Error: install(EXPORT "AMR-WindTargets" ...) includes target "amrwind_obj" which requires target "amrex_hydro" that is not in any export set.

Reply:
Ok. Thanks for confirming